src/scenes/Event/store.js

```diff
@@ -200,12 +205,12 @@ export default class EventStore {
 
   @action
   verifyConfirmedOracle = async () => {
-    // TODO: need to implement hashedId on server first so we can fetch the updated Oracle by hashedId as it goes through the approve -> createEvent txs
-    // const res = await queryAllOracles([{ txid: this.txid }]);
-    // const { topicAddress, address, txid } = res[0];
-    // if (address && topicAddress) {
-    //   this.app.router.push(`/oracle/${topicAddress}/${address}/${txid}`);
-    // }
+    const res = await queryAllOracles([{ hashId: this.hashId }]);
+    const confirmedOracle = _.filter(res, 'topicAddress');
```

Comment:
why do we need to filter? there should only be 1 returning?

Reply:
this is used to handle the `approve` tx, since this time the oracle doesn't have topicAddress in it, so we should not push the new path.

Reply:
hm? I'm not understanding this. The `queryAllOracles` res should have only 1 Oracle obj in the array. So we don't need to `filter` the 1 item array.